NEWS.md

```diff
@@ -538,8 +540,8 @@ Deprecated or removed
     argument specifying the floating point type to which they apply. The old
     behaviour and `[get/set/with]_bigfloat_rounding` functions are deprecated ([#5007])
 
-  * `cholpfact` and `qrpfact` are deprecated in favor of keyword arguments in
-    `cholfact(..., pivot=true)` and `qrfact(..., pivot=true)` ([#5330])
```

Comment:
This is in the Julia 0.3 release notes. I think we need a separate entry for these changes under Julia 0.4. Also it would be good to have deprecations in `deprecated.jl`.